Managing/budget-manager.js

```diff
@@ -258,11 +240,12 @@ function makeCampaignFilterStatements(campaignNameContains, campaignNameDoesNotC
 }
 
 
-function combineQueries(dates, campaignFilterQueries) {
+function combineQueries(campaignFilterQueries) {
```

Comment:
What do you think of naming this function differently, now we're not combining with dates?